sentinel-extension/sentinel-logging-extension-slf4j/README.md

```diff
@@ -0,0 +1,49 @@
+# Sentinel Logging Extension SLF4J
+
+To use Sentinel Logging Extension SLF4J with Log4j2, you should add the following dependency firstly:
+
+```xml
+<dependency>
+    <groupId>com.alibaba.csp</groupId>
+    <artifactId>sentinel-logging-extension-slf4j</artifactId>
+    <version>x.y.z</version>
+</dependency>
+<dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-api</artifactId>
+    <version>${slf4j.version}</version>
+</dependency>
```

Comment:
> Coming to the usage we may not force user to introduce both of them because:
> 
> 1. User should have already had a configured slf4j logging system and just want sentinel's logs merged into.

I mark the slf4j dependence with socpe "provided" in pom.xml. The dependence should be providered by user



> ... and surely i am not sure whether we should provide some guidances on how to integrate slf4j over log4j2.

In README.md i provider the loging's configuration just want to emphasize:
> And you can add Sentinel's Loggers that it name is `sentinelRecordLogger` or `sentinelCommandCenterLogger` for your needs